src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -430,43 +413,36 @@ private void createTrayItem()
         // set up hide-to-tray behavior
         trayItem.addSelectionListener(new SelectionListener()
         {
-            @Override
             public void widgetSelected(SelectionEvent e)
             {
                 shell.setVisible(true);
                 shell.setFocus();
             }
 
-            @Override
             public void widgetDefaultSelected(SelectionEvent e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)